companies/1und1-gmx.json

```diff
@@ -17,7 +17,7 @@
         "GMX Tresor",
         "GMX Internet Browser"
     ],
-    "address": "c/o 1&1 Mail & Media GmbH\nElgendorfer Straße 57\n56410 Montabaur\nDeutschland",
+    "address": "c/o 1&1 Mail & Media GmbH , Elgendorfer Straße 57,56410 Montabaur, Deutschland",
```

Comment:
🚫 **[recordlint]** <[end-address-in-country](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Last line of `address` (`c/o 1&1 Mail & Media GmbH , Elgendorfer Straße 57,56410 Montabaur, Deutschland`) is not a country.

We've decided on specific variations for these countries:
* `United States of America`
* `The Netherlands`
* `Republic of Singapore`